source/meta.tex

```diff
@@ -1425,8 +1425,8 @@
  \tcode{struct is_nothrow_constructible;}   &
  \tcode{is_constructible_v<T,} \tcode{ Args...>} is \tcode{true}
  and the
- variable definition for \tcode{is_constructible}, as defined below, is known not to
- throw any exceptions\iref{expr.unary.noexcept}.
+ variable definition for \tcode{is_constructible}, as defined below,
+ contains no potentially-throwing expression\iref{except.spec}.
```

Comment:
What does "contains" mean?  The variable definition itself is not an expression.